windows/src/engine/keyman32/build.sh

```diff
@@ -7,50 +7,80 @@ THIS_SCRIPT="$(readlink -f "${BASH_SOURCE[0]}")"
 
 builder_describe "Keystroke processing engine (32 bit)" \
   @/common/include \
-  @/core:x86 \
-  clean configure build test publish install
+  @/core \
+  clean configure build test publish install \
+  :x86 :x64
 
 builder_parse "$@"
 
 #-------------------------------------------------------------------------------------------------------------------
 
 source "$KEYMAN_ROOT/resources/build/win/environment.inc.sh"
 WIN32_TARGET="$WIN32_TARGET_PATH/keyman32.dll"
+X64_TARGET="$X64_TARGET_PATH/keyman64.dll"
 
 builder_describe_outputs \
-  configure:project    /resources/build/win/delphi_environment_generated.inc.sh \
-  build:project        /windows/src/engine/keyman32/$WIN32_TARGET
+  configure    /resources/build/win/delphi_environment_generated.inc.sh \
+  build:x86    /windows/src/engine/keyman32/$WIN32_TARGET \
+  build:x64    /windows/src/engine/keyman64/$X64_TARGET \
 
 #-------------------------------------------------------------------------------------------------------------------
 
-function do_clean() {
-  vs_msbuild keyman32.vcxproj //t:Clean
+function do_clean_x86() {
+  vs_msbuild keyman32.vcxproj //t:Clean //p:Platform=Win32
   clean_windows_project_files
 }
 
-function do_build() {
+function do_clean_x64() {
+  vs_msbuild keyman32.vcxproj //t:Clean //p:Platform=x64
+  clean_windows_project_files
+}
+
+function do_build_x86() {
   create-windows-output-folders
   build_version.res
   vs_msbuild keyman32.vcxproj //t:Build "//p:Platform=Win32"
   cp "$WIN32_TARGET" "$WINDOWS_PROGRAM_ENGINE"
   cp "$WIN32_TARGET_PATH/keyman32.pdb" "$WINDOWS_DEBUGPATH_ENGINE"
 }
 
-function do_publish() {
+function do_build_x64() {
+  create-windows-output-folders
+  run_in_vs_env rc version64.rc
+  vs_msbuild keyman32.vcxproj //t:Build "//p:Platform=x64"
+  cp "$X64_TARGET" "$WINDOWS_PROGRAM_ENGINE"
+  cp "$X64_TARGET_PATH/keyman64.pdb" "$WINDOWS_DEBUGPATH_ENGINE"
+}
+
+function do_publish_x86() {
   wrap-signcode //d "Keyman Engine for Windows" "$WINDOWS_PROGRAM_ENGINE/keyman32.dll"
   wrap-symstore "$WINDOWS_PROGRAM_ENGINE/keyman32.dll" //t keyman-engine-windows
   wrap-symstore "$WINDOWS_DEBUGPATH_ENGINE/keyman32.pdb" //t keyman-engine-windows
 }
 
-function do_install() {
+function do_publish_x64() {
+  wrap-signcode //d "Keyman Engine for Windows" "$WINDOWS_PROGRAM_ENGINE/keyman64.dll"
```

Comment:
Do we want to distinguish x86 and x64 in the description?

Reply:
Not necessary here -- this is a description that will be visible only in the code signature details, which almost no one will ever see, particularly for a .dll.